specification/search/data-plane/Azure.Search/readme.md

```diff
@@ -100,10 +99,21 @@ vararg-properties: >-
   SuggestOptions.orderBy, SuggestOptions.searchFields, SuggestOptions.select
 
 directive:
+    # Enable configuration of RestProxy serializer for DocumentsImpl
```

Comment:
We need to be using `JacksonAdapter` here, the codegen models don't support other serializers so this may cause issues.